src/core/namespace.c

```diff
@@ -202,7 +204,17 @@ static const MountEntry protect_home_yes_table[] = {
 
 /* ProtectControlGroups=yes table */
 static const MountEntry protect_control_groups_yes_table[] = {
-        { "/sys/fs/cgroup",      MOUNT_READ_ONLY,     true  },
+        { "/sys/fs/cgroup",      MOUNT_READ_ONLY,                   false  },
```

Comment:
The change `true` -> `false` is intentional ?

Reply:
So yea it is because this sets ignore = false which means fail hard if the target mount path doesn't exist. This was existing behavior prior to this PR so I was keeping with it...but I forgot to set it to false in the first commit. Let me fix that!